docs/kubernetes/arkmanager/configmap.yaml

```diff
@@ -188,10 +188,10 @@ data:
     ResourcesRespawnPeriodMultiplier=0.5
     alwaysNotifyPlayerJoined=True
     alwaysNotifyPlayerLeft=True
-    TamingSpeedMultiplier=3.00000

```

Comment:
I think taming speed is ok, the problem is more about we only taming one creature high level once, our breeding is bit OP, maybe we can make breeding, or hatching longer (of course has to be without S+)